libraries/standard/mqtt/cbmc/stubs/network_interface_stubs.c

```diff
@@ -72,7 +72,7 @@ int32_t NetworkInterfaceSendStub( NetworkContext_t * pNetworkContext,
      * iterations. Looping for INT32_MAX times adds no value to the proof.
      * What matters is that the MQTT library can handle all the possible values
      * that could be returned. */
-    if( tries < MAX_NETWORK_SEND_TRIES )
+    if( tries < ( MAX_NETWORK_SEND_TRIES - 1 ) )
```

Comment:
Unwinding assertion flags have been added to be run in the CI, so this line is fixed to be within the exclusive tries bound.